oshi-core/src/main/java/oshi/hardware/CentralProcessor.java

```diff
@@ -165,6 +165,42 @@ public interface CentralProcessor {
      */
     double[] getSystemLoadAverage(int nelem);
 
+    /**
+     * This is a convenience method which collects an initial set
+     * of ticks using {@link #getSystemCpuLoadTicks()} and passes
+     * that result to {@link #getSystemCpuLoadBetweenTicks(long[])}
+     * after the specified delay.
+     *
+     * @param delay
+     *          Milliseconds to wait.
+     * @return value between 0 and 1 (100%) that represents the cpu usage in the provided time period.
+     */
+    default double getSystemCpuLoad(long delay) {
+        long start = System.nanoTime();
+        long[] oldTicks = getSystemCpuLoadTicks();
+        long toWait = delay - (System.nanoTime() - start) / 1_000_000;
+        if(toWait > 0L) Util.sleep(delay); // protect against IllegalArgumentException
```

Comment:
(Style) Please use curly braces and indent the result of this conditional.  Checkstyle will probably already be telling you this.